### PR TITLE
Add a better message if the user doesn't specify a resource type.

### PR DESCRIPTION
**Pull request by brendandburns**
Closes https://github.com/GoogleCloudPlatform/kubernetes/issues/8013

### CODE REVIEW

File: pkg/kubectl/cmd/get.go [hunk 1]
```diff
@@ -94,6 +95,23 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		return err
 	}
 
+	if len(args) == 0 {
+		fmt.Fprint(out, `
+You must specify the type of resource to get. Valid resource types include:
+   * pods (aka 'po')
+   * replicationcontrollers (aka 'rc')
+   * services
```
Comment (hurf) [inline pkg/kubectl/cmd/get.go]: (aka 'svc')

Comment (brendandburns) [inline pkg/kubectl/cmd/get.go]: this is marked as 'deprecated will remove before 1.0' in https://github.com/GoogleCloudPlatform/kubernetes/blob/5520386b180d3ddc4fa7b7dfe6f52642cc0c25f3/pkg/kubectl/kubectl.go#L115 so I didn't include it

**Comment by brendandburns** [comment]
Comment addressed, please take another look.

**Comment by brendandburns** [comment]
@goltermann @juliaferraioli @kprobst

File: pkg/kubectl/cmd/get.go (hunk 1 above)
Comment (hurf) [inline pkg/kubectl/cmd/get.go]: Oh, I thought the deprecation was only for 'se'

Comment (vreon) [inline pkg/kubectl/cmd/get.go]: @hurf yeah, looking at the commit where the comment was added (https://github.com/GoogleCloudPlatform/kubernetes/commit/09dcec57fb6a2956b815efbd448afea75529a29a), it seems like that was the intent

**Pull request merged**